package-lock.json

```diff
@@ -91,7 +91,7 @@
         "string-width": {
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/string-width/-/string-width-2.1.1.tgz",
-          "integrity": "sha512-nOqH59deCq9SRHlxq1Aw85Jnt4w6KvLKqWVik6oA9ZklXLNIOlqg4F2yrT1MVaTjAqvVwdfeZ7w7aCvJD7ugkw==",
+          "integrity": "sha1-q5Pyeo3BPSjKyBXEYhQ6bZASrp4=",
```

Comment:
Erg, this is an artefact of an old and broken version of npm being used, or at least cached files downloaded by such a version. Update npm to latest, then follow https://github.com/npm/npm/issues/17749#issuecomment-322627101 to fix it, revert the changes to `package-lock.json`, and do an `npm i` again.